src/zenml/cli/example.py

```diff
@@ -638,6 +639,15 @@ def run(
     # TODO [ENG-272]: - create a post_run function inside individual setup.sh
     #  to inform user how to clean up
     examples_dir = Path(os.getcwd()) / path
+
+    if sys.platform == "win32":
+        logger.info(
+            "If you are working on Windows, make sure that you have an "
+            "associated application with executing .sh files. If you don't "
+            "have any and you see a pop-up during 'zenml pipeline run', we "
+            "suggest to use the GitBash."
```

Comment:
I would say "we suggested using [Git BASH](https://gitforwindows.org/)

Reply:
Thanks. Updated the message.